src/core/runtime-config.ts

```diff
@@ -0,0 +1,47 @@
+import { snakeCase } from 'scule'
+import { startServer } from './server'
+
+const NUXT_ENV_PREFIX = 'NUXT_'
```

Comment:
While not fully documented, it might be worth to not hardcode it (using it as default is fine) but rely on the nitro runtime config value. See https://github.com/nuxt/nuxt/issues/23714

Reply:
Oh, I haven't seen that being used before, I've changed the function to accept an optional argument `envPrefix`.

>... rely on the nitro runtime config value

It would be nice if we could retrieve which prefix is being used in the running fixture, I'm not sure how to do that. An alternative would be checking if `nuxtConfig.runtimeConfig.nitro.envPrefix` is present in the test context and using that.

Reply:
yeah, I think the latter would be the easiest, defaulting to `NUXT_` otherwise